src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -425,6 +436,28 @@ private static int LineBreaksAfterSemicolon(SyntaxToken currentToken, SyntaxToke
             {
                 return 0;
             }
+            else if (currentToken.Parent is PropertyDeclarationSyntax property)
+            {
+                // If the current semicolon token belongs to a property
+                // then it is a semicolon at the end of a property typically (but not always) after a property initializer:
+                // public int Prop { get; } = 1; <-- this one
+                // public int Prop { get; }; <-- this produces a syntax error, but the semicolon is still attached to the property
+                // In such cases we need to have 2 line breaks in order to have proper separation between members of a class, struct etc.
+                // The only exception is when the next token starts a new single-line property.
+                // In such case we want to group these properties together by having only 1 line break.
+                // Note: case, when the property is the last member and needs only 1 line break after it is handled above (the next token is a closing brace then)
+                Debug.Assert(((PropertyDeclarationSyntax)currentToken.Parent).SemicolonToken == currentToken);
+
+                if (IsSingleLineProperty(property) &&
+                    nextToken.Parent is PropertyDeclarationSyntax)
```

Comment:
This doesn't seem correct. Please add a test that has a single-line property followed by a manually-implemented property.